src/components/AssetsTable/index.jsx

```diff
@@ -386,44 +405,19 @@ export class AssetsTable extends React.Component {
     );
   }
 
-  /**
-   * Rendering an empty status alert keeps the alert in the DOM initially and
-   * enables screen readers to listen for changes.
-   */
-  renderEmptyStatusAlert() {
-    return (
-      <div>
-        <StatusAlert
-          dialog=""
-          open={false}
-          dismissible={false}
-        />
-      </div>
-    );
-  }
-
   renderStatusAlert() {
     const { assetsStatus } = this.props;
-    const deleteActions = [assetActions.DELETE_ASSET_FAILURE, assetActions.DELETE_ASSET_SUCCESS];
-    const uploadActions = [assetActions.UPLOAD_ASSET_SUCCESS,
-      assetActions.UPLOAD_EXCEED_MAX_COUNT_ERROR, assetActions.UPLOAD_EXCEED_MAX_SIZE_ERROR,
-      assetActions.UPLOAD_ASSET_FAILURE, assetActions.UPLOADING_ASSETS];
-    let status = {};
+
     let onClose = this.closeStatusAlert;
-    if (assetsStatus.type === assetActions.TOGGLING_LOCK_ASSET_FAILURE) {
-      status = this.getLockFailedStatusFields(assetsStatus);
-    } else if (deleteActions.includes(assetsStatus.type)) {
-      status = this.getDeleteStatusFields(assetsStatus);
+    if (this.deleteActions.includes(assetsStatus.type)) {
```

Comment:
I know that we always make a getAssets call on every page load, so there is always some kind of assetsStatus state. But the default assetsStatus state is {}. Is there any reason we'd get into a position where assetsStatus is {}? If our first getAssets times out, would it remain {}? I only ask because then assetsStats.type would be undefined, and I'm not sure what would happen in that case. Should we add something to be a little more robust to that? Or should we just let it go, since we haven't run into that yet?

Reply:
See comment below -- I would want to sanity check the `undefined`/`null` cases there.

Reply:
Update: calling `hasOwnProperty()` with `null` or `undefined` simply returns `false`.  I tested with the built in area here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty. Here's the link for the `in` operator as well for reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in - it looks like we can leverage `in` with Arrays as well, but I think in this case, updating to use the constants objects if we can as well as `hasOwnProperty` to validate on the top level is the best route if we make a change here.